fix(jans-cli-tui): fix  zipapp build for ubuntu

```
  Downloading idna-3.4-py3-none-any.whl (61 kB)
Collecting charset-normalizer~=2.0.0
  Downloading charset_normalizer-2.0.12-py3-none-any.whl (39 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)
Building wheels for collected packages: jans-cli-tui, pyDes
  Building wheel for jans-cli-tui (pyproject.toml): started
  Building wheel for jans-cli-tui (pyproject.toml): finished with status 'done'
  Created wheel for jans-cli-tui: filename=jans_cli_tui-1.0.11_SNAPSHOT-py3-none-any.whl size=195437 sha256=0b31670a7adf8b42b78373ce2b7b24f5c8b66b484f04c5169dd31a30a3ba5fcd
  Stored in directory: /tmp/pip-ephem-wheel-cache-6vdnyvgv/wheels/f1/70/84/4b4f5f7dc8f21938f9e01fe0b4094d8c856819214a846c9988
  WARNING: Built wheel for jans-cli-tui is invalid: Metadata 1.2 mandates PEP 440 version, but '1.0.11-SNAPSHOT' is not
  Building wheel for pyDes (setup.py): started
  Building wheel for pyDes (setup.py): finished with status 'done'
  Created wheel for pyDes: filename=pyDes-2.0.1-py2.py3-none-any.whl size=9549 sha256=a2bb2ce748dfcf9dcd4c6e74821b0cf085635a99281b9304fca93cb8d3fc9216
  Stored in directory: /tmp/pip-ephem-wheel-cache-6vdnyvgv/wheels/a9/8b/1e/0c2835e2802d6f48e1c3dbb0335fa62f730e61fdf963ab3827
Successfully built pyDes
Failed to build jans-cli-tui
ERROR: Could not build wheels for jans-cli-tui, which is required to install pyproject.toml-based projects

Pip install failed!

make: *** [Makefile:13: zipapp] Error 1
```

To reproduce checkout the latest and cd into `jans-cli-tui`  then `make zipapp`

--- a/jans-cli-tui/setup.py
+++ b/jans-cli-tui/setup.py
@@ -60,6 +60,7 @@ def find_version(*file_paths):
     here = os.path.abspath(os.path.dirname(__file__))
     with codecs.open(os.path.join(here, *file_paths), 'r') as f:
         version_file = f.read()
+    version_file = version_file.replace('-SNAPSHOT', '-dev')
     version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
                               version_file, re.M)
     if version_match:

